src/main/storage/storage.ts

```diff
@@ -100,6 +100,9 @@ export function writeDirectoryConfig(directory: string, cfg: Config): Promise<vo
   const cfgString = JSON.stringify(cfg);
 
   return fs.writeFile(file, cfgString)
+    .then(() => {
+      return fs.truncate(file, Buffer.byteLength(cfgString));
```

Comment:
I kinda remember removing this line for some reason,  @CoreyGair ?

Reply:
that was me because it corrupted .bones files when saving recordings at the same time (which is done for the two separate audio and video files)